relay-general/src/protocol/event.rs

```diff
@@ -67,13 +67,14 @@ impl_str_serde!(EventId);
 #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, Deserialize, Serialize)]
 #[serde(rename_all = "snake_case")]
 pub enum EventType {
-    Default,
     Error,
     Csp,
     Hpkp,
     ExpectCT,
     ExpectStaple,
     Transaction,
+    #[serde(other)]
+    Default,
```

Comment:
Since we deserialize this in the endpoint now, we make sure that we don't accidentally error out if there is an unknown event type, especially for external Relays. Normalization corrects this in `FromValue`.